src/piper.js

```diff
@@ -5,10 +5,11 @@ const fs = require('fs')
 
 async function run () {
   try {
+    const command = core.getInput('command', { required: true })
```

Comment:
I was not aware of this option. How does it relate to this flag? https://github.com/SAP/project-piper-action/blob/master/action.yml#L6

Reply:
I was not aware of the `yaml` 😉 

The required flag makes the `core.getInput` fail if the parameter is not set.

Reply:
I was not aware of the `yaml` 😉 

The required flag makes the `core.getInput` fail if the parameter is not set.

Reply:
but with the provided default (in the yaml file), I'm not sure how that can happen. And I think `help` is a sensible default.

Reply:
So I'm not sure about the benefit of this change, but also I don't see any drawbacks currently..

Reply:
As the yaml goes into the `dist/index.js` I guess we would need to test this with an integration test. This way we have a test next to the code. But on the other hand, why build it again if we already get it for free..

Reply:
Or lets see this as a fallback.